camptix.php

```diff
@@ -2800,7 +2802,6 @@ function generate_attendee_report( $format ) {
 			'order' => 'ASC',
 			'cache_results' => false,
```

Comment:
I wonder if this is part of the problem? If this were `true` then the `get_post_meta()` call would fetch from memory instead of hitting the database. If that's true, then we wouldn't need the memoization.

I'm not sure why it was originally added, though, so flipping it to `true` might have side-effects. It might have something to do with the "magic" code that the comments refer to.

Possibly related: If we switched to using `WP_Query` directly, [we'd have some more control over performance](https://10up.github.io/Engineering-Best-Practices/php/).

Reply:
I'm not sure, but I have a feeling the "magic" code may be over-engineering/premature-optimization, and we could safety refactor it to use a more traditional fetch. That may or may not solve the problem here, but it seems like it'd help by eliminating the db queries for metadata. It'd also be nice since it'd make the code a lot simpler.

Reply:
Unless I need to do more than just flipping the bit. On the get go it doesn't look like metadata calls are getting fetched from db, but I guess I taking a deeper look here won;t hurt.

Reply:
Hmm, I would think just flipping that would be enough to test, but maybe the "magic" code or something else is interfering. You can use `define( 'SAVEQUERIES', true );` to see what queries are being made.